css/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+	font-family: "Segoe UI", Arial, sans-serif;
+	background: #f1f1f1;
+	color: #5f5f5f;
+}
+
+header {
+	text-align: center;
+}
+
+.nameInput {
```

Comment:
you can apply this style for all inputs elements using just `input` tag. Since normally all inputs on the page should be the same height.

Reply:
Thank you for advice, fixed.